web/modules/custom/brms_common/brms_common.deploy.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ * Deploy functions run after drush config:import
+ */
+
+use Drupal\migrate\MigrateExecutable;
+use Drupal\migrate\MigrateMessage;
+
+/**
+ * Initialise taxonomy terms.
+ */
+function brms_common_deploy_001(&$sandbox = NULL) {
```

Comment:
I like the idea of numbering the deploys but probably should be better to do `brms_common_deploy_001_short_description`? I'll add an adr proposal